rust/repositories.bzl

```diff
@@ -12,6 +14,33 @@ def rust_repositories():
 
     RUST_VERSION = "1.35.0"
 
+    maybe(
+        http_archive,
+        name = "bazel_skylib",
```

Comment:
How did this work before? Should there be some equivalent deletion?

It would also be good to pull out a `def bazel_skylib_repositories` to clarify that _we_ don't depend on rules_python directly (do we?)

Reply:
I don't believe we need all that. We only need to compile C++ proto compiler everything else is in rust, why would we need the python defs?

Reply:
We need rules_proto and skylib for building protobuf. I've moved those 2 repos to //proto:repositories.bzl, you're right that they belong there.